cmd/kms-handlers.go

```diff
@@ -223,6 +230,22 @@ func (a kmsAPIHandlers) KMSDeleteKeyHandler(w http.ResponseWriter, r *http.Reque
 // KMSListKeysHandler - GET /minio/kms/v1/key/list?pattern=<pattern>
 func (a kmsAPIHandlers) KMSListKeysHandler(w http.ResponseWriter, r *http.Request) {
 	ctx := newContext(r, w, "KMSListKeys")
+	if GlobalKMS != nil {
+		if GlobalKMS.IsLocal() {
+			secretKey := GlobalKMS.ListLocal()
```

Comment:
this is not a secretKey since we use secretKey to designate `access credentials` - perhaps call this `kmsSecret` ?